CHANGELOG.md

```diff
@@ -4,24 +4,39 @@ Changes from v1.4 to v1.5
 API Changes
 -----------
 
+- `drawImage()` now accepts `exptime` and `area` keywords to indicate the image
+  exposure time and telescope collecting area. (#789)
+- dimensions of `SED` changed from [photons/wavelength-interval] to either
+  [photons/wavelength-interval/area/time] or [1] (dimensionless).  
+  `ChromaticObject`s representing stars or galaxies take SEDs with the former
+  dimensions, those representing a chromatic PSF take SEDs with the latter
+  dimensions. (#789)
+- Added restrictions to `ChromaticObject`s and `SED`s consistent with
+  dimensional analysis.  E.g., only `ChromaticObject`s with dimensionful SEDs
+  can be drawn. (#789)
 - Simplified the return value of galsim.config.ReadConfig. (#580)
 
 
 Dependency Changes
 ------------------
-
+- `astropy` is now a required dependency instead of an optional one. (#789)
```

Comment:
Required for all GalSim functionality?  Or just chromatic functionality? Either way, I think an update in INSTALL.md is needed to reflect this.